### DISCUSSION TITLE
New G7 isn't detected

### DISCUSSION THREAD

**Discussion by BrianAllred** [NightscoutFoundation/xDrip#3548] [Q&A] [answered]
Pixel Fold
Latest xDrip pre-release
Worked fine with previous G7

My t.slim X2 said my old G7 errored out and to replace it, so I did.

I put the old sensor in the microwave and put the new code in xDrip. It still tried to connect to the old sensor a couple of times (mismatch), so I removed the Bluetooth pairing and moved farther away from the microwave.

Now xDrip just sits on scanning forever. I tried rebooting the phone, I tried toggling Bluetooth, I tried clearing the code, I tried changing collection mode to no collection, then back to G7. Nothing has worked.

The new sensor is communicating with my pump just fine.

**Comment by Navid200** [comment]
So, you have readings in t:connect?

Can you post a screenshot of the Dex status page?

**Reply by BrianAllred** [reply]
We got a little farther. Looks like it's trying to connect occasionally, but it's still never successful.

![image](https://github.com/NightscoutFoundation/xDrip/assets/5847511/3bc46c39-8279-44df-8998-46980bb7ce3c)

![image](https://github.com/NightscoutFoundation/xDrip/assets/5847511/680ec0b0-f9ac-48a9-a1c4-daee5f9ef8dd)

**Reply by Navid200** [reply]
I don't know what the first screenshot is.
You have cut out the top of the screenshots.  So, I cannot see if you have Dexcom app installed.

If you need help, you should provide clear information here.
I want to help you if you need it.

Everything is explained here.  I hope you can sort it out:
https://navid200.github.io/xDrip/docs/Connectivity-troubleshoot.html

**Reply by BrianAllred** [reply]
The first screenshot is the t:connect app.

I didn't cut out anything, that's what screenshots looks like. I'm not trying to hide anything or make anything difficult. I felt like I was being clear, but if I'm not, and I need to clarify anything, please let me know.

I thought you wanted a screenshot of the xDrip status page, but maybe you're talking about the status page from the Dexcom app? In which case, I don't have that installed. But I can install it if you need me to.

**Reply by Navid200** [reply]
Do you have Dexcom app installed?

**Reply by BrianAllred** [reply]
No.

**Reply by Navid200** [reply]
Can you post a screenshot of xDrip logs?
https://navid200.github.io/xDrip/docs/Logs.html

**Reply by BrianAllred** [reply]
I moved around the house a bit and I guess it picked up my old sensor a couple of times. But I've kept the screen open, volume is up, and when the old sensor is out of range, the app just scans and scans.

![Screenshot_20240627-152504](https://github.com/NightscoutFoundation/xDrip/assets/5847511/192cfb11-9cce-4b83-9189-61dd98cc092e)

**Reply by Navid200** [reply, marked as answer]
Do you mind enabling detail logs and letting it run for may be 15 minutes and then uploading the logs?
https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html  

Do you have Nightscout or Tidepool set up?  I hope so.  Then, you have all your history backed up in the cloud.

Regardless, create a backup: https://navid200.github.io/xDrip/docs/Backup.html  

Then, making sure you have created a backup, uninstall xDrip.  Then, install the latest Nightly again.  
It will tell you that it has detected a backup asking you if you want to restore it.  Respond no.  

Then, follow everything under the heading establishing connectivity here: https://navid200.github.io/xDrip/docs/Dexcom/G7.html
After everything works and you have 3 readings, go ahead and restore the backup.

**Reply by BrianAllred** [reply]
I uploaded the logs, reinstalled, got it set up and finally got a pairing request. Once I had a few readings, I restored from backup, backfilled to Nightscout, and everything is running great again.

Thanks for the help!

**Reply by Navid200** [reply]
You are welcome.
Thanks for reporting this.
I am sorry about the misunderstanding.  
Hopefully, we can sort this out so that it won't happen again.

**Comment by jamorham** [comment]
Quadruple check that you have the transmitter pairing code correct for this new transmitter. Try also changing it to something else then changing it back again.

**Reply by Navid200** [reply]
I suspect he is not hearing the pair request ping.  Either the volume is lowered to 0 or the phone is muted.  That's why I want to see the logs to see if there has been a pair request or not.

**Reply by BrianAllred** [reply]
As stated, I've tried clearing the code and re-entering it as well as disabling collection altogether. The xDrip scanning screen has been up and my volume is on.
